release/README.DOCKER.md

```diff
@@ -1,36 +1,35 @@
-# BUX Wallet Frontend
+# SPV Wallet Frontend
 
-The `bux-wallet-frontend` is a referential frontend application for a custodial web wallet. 
-It is used in conjunction with the `bux-wallet-backend` component, which serves as the backend server. The frontend is written in React and can be run as a Docker image named `${DOCKERHUB_OWNER}/${DOCKERHUB_REPO}`, exposed on port 3002.
+The `spv-wallet-web-frontend` is a referential frontend application for a custodial web wallet.
+It is used in conjunction with the `spv-wallet-web-backend` component, which serves as the backend server. The frontend is written in React and can be run as a Docker image named `${DOCKERHUB_OWNER}/${DOCKERHUB_REPO}`, exposed on port 3002.
```

Comment:
Same as in Readme, 
I you don't mind, I would like to ask you: could you add some information that backned is also using spv-wallet, and changing it's wallet nature from non-custodial to custodial. 🙏

Reply:
I will put here the same section as I proposed in the comment above.

Reply:
Done.